_includes/header.html

```diff
@@ -0,0 +1,37 @@
+<header class="header">
+	<div class="header_content">
+		<div class="header_logo">
+			<a href="{{ site.url }}/{{ page.lang }}" class="header_logo_link">
+				{% include logo_drawings.svg %}
+				{% include logo_txt.svg %}
+			</a>
+		</div>
+		<div class="header_navs">
+			<nav class="header_mainnav">
+				<ul class="header_mainnav_items">
+					{% for navbar_item in page.navbar_items %}
+						<li class="header_mainnav_item">
+							{{ navbar_item }}
+						</li>
+					{% endfor %}
+				</ul>
+			</nav>
+			<nav class="header_langnav">
+				<ul class="header_langnav_items">
+					<li class="header_langnav_item">
+						<a href="/fr" class="header_langnav_link">
+							<img class="header_langnav_flag" src="/assets/img/flags/fr.svg" alt="Passer au français">

```

Comment:
Redundant with the text.
```suggestion
							<img class="header_langnav_flag" src="/assets/img/flags/fr.svg" alt="">

```